packages/navigator/package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "@theia/core": "^0.3.7",
     "@theia/filesystem": "^0.3.7",
-    "@theia/workspace": "^0.3.7"
+    "@theia/workspace": "^0.3.7",
+    "@theia/editor": "^0.3.7"
```

Comment:
As discussed, we do not want to introduce a dependency for that.

Reply:
ok. Just to clarify things for me. We do not introduce this dependency to be able to introduce other editors/viewers (for example for images) and have ability to link them with the navigator. Is it the main reason or we just tend to have everything decoupled as much as possible?

Reply:
Both. Does it answer your question or you have expected a priority between the two reasons?

Reply:
Yes this answers my question. Thank you.